internal/tools/orchestrator/services/addon/addon.go

```diff
@@ -3072,6 +3149,58 @@ func (a *Addon) deployAddons(req *apistructs.AddonCreateRequest, deploys []dbcli
 	return nil
 }
 
+func (a *Addon) nacosVersionReference(version string) (reg, conf string, err error) {
+	regMap, err := GetCache().Get(RegisterCenterAddon)
+	if err != nil {
+		return "", "", err
+	}
+	confMap, err := GetCache().Get(ConfigCenterAddon)
+	if err != nil {
+		return "", "", err
+	}
+	registers, ok := regMap.(*VersionMap)
+	if !ok {
+		return "", "", errors.New("cache data type error")
+	}
+	configs, ok := confMap.(*VersionMap)
+	if !ok {
+		return "", "", errors.New("cache data type error")
+	}
+	for _, r := range *registers {
```

Comment:
Code redundancy. Similar logic for registercenter and configcenter.
Assert target all is `VersionMap`.